To-List-Project-HTML2.html

```diff
@@ -4,229 +4,30 @@
-					this.style.opacity = 1;
-					}
-				)
-				.mouseout(function(event) {
-					this.style.opacity = 0.1;
-					}
-				)
-				.on("click", function(event) {
-					event.preventDefault();
-					$thisObject = $(this);
-					$(".subForm").remove();
-					makeSubForm();
-					}
-				)
-			
-			//relocate mainForm and mainListAddIcon to the bottom of the list
-			$mainForm.appendTo(".mainList");
-			var mainListItem = blankItem.text(inputAnswer).append($mainListItemAddIcon, blankList);	
-			mainListItem.appendTo(".mainList");
-			console.log("mainList item created. class name= " + blankList.attr("class"));
-			parentListIndex ++;
-		}
-			
-		//When mainListItemAddIcon is clicked, append subForm to corresponding mainListItem .
-		var makeSubForm = function(event) {
-			console.log("clicked add sub list item");
-			
-			//Prepare a subForm which adds a new subListItem item (li) to the appropriate place when submitted
-			var index = $thisObject.attr("class").substr($thisObject.attr("class").length - 1);
-			console.log("subForm index = " + index);
-			var subForm = new form();
-			var $subForm = $("<form>").attr("class", "subForm").append(subForm.label).append(subForm.button)
-				.on("submit", function(event) {
-					event.preventDefault();
-					inputAnswer = $(this).find("input").val();
-					makeSubListItem();
-					}
-				)
-			console.log("subform generated: ") 
-			console.log($subForm);
-			
-			var indexString = String(".item" + index);
-			$subForm.appendTo($(indexString));	
-		}
-		
-		var makeSubListItem = function(event) {
-			var blankItem = $("<li>");
-			var index = $thisObject.attr("class").substr($thisObject.attr("class").length - 1);
-			var indexString = String(".item" + index);
-			console.log("created subListItem: " + indexString);
-			var subListItem = blankItem.text(inputAnswer);
-			$(indexString).append($("<li>").html(subListItem));	
-		}
-		
-		//MAIN CODE
-		$mainListAddIcon.appendTo(".mainList");
-		
-		$(".removeForms").on("click", function(event) {
-			event.preventDefault();
-			$(".subForm").remove();
-			$(".form").remove();
-			console.log("removed forms");
-			}
-		)
-		
-		$(".save").on("click", function(event) {
-			event.preventDefault();
-			
-			var fieldValue = $(".lists").html();
-			console.log(fieldValue);
-			localStorage.setItem("everything", fieldValue);
-			console.log("saved work");
-			}
-		)
-		
-		var load = function() {
-			console.log("attempting load....")
-			var storedValue = localStorage.getItem("everything");
-			if (storedValue) {
-				$(".lists").html(storedValue);
-				console.log("found file. load successful");
-				return;
-			}
-			console.log("nothing is in storage");
-		}
-
-    </script>
-	
-</body>
-
 
+	<br>
 
+	<div class= dialog></div>
 
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+  <script src="todo.js" type="text/javascript"></script>
```

Comment:
yep like the episode on Silicon Valley...

Some people prefer tabs but a lot of people including me prefer spaces because it renders the same on every computer whereas tabs in some computers will be 2, in others 4, and in some others 8